platform-includes/capture-error/flutter.mdx

```diff
@@ -1,4 +1,32 @@
 <PlatformContent includePath="capture-error/dart.mdx" />
 
-- Flutter-specific errors, such as using `FlutterError.onError`, are captured automatically
-- The SDK already runs your init `callback` on an error handler, such as [`runZonedGuarded`](https://api.flutter.dev/flutter/dart-async/runZonedGuarded.html) on Flutter versions prior to `3.3`, or [`PlatformDispatcher.onError`](https://api.flutter.dev/flutter/dart-ui/PlatformDispatcher/onError.html) on Flutter versions 3.3 and higher, so that errors are automatically captured.
+### FlutterError.onError
+
+Flutter-specific errors, such as using `FlutterError.onError`, are captured automatically.
+
+### PlatformDispatcher.onError / runZonedGuarded
+
+The SDK already runs your init `callback` on an error handler, such as [`runZonedGuarded`](https://api.flutter.dev/flutter/dart-async/runZonedGuarded.html) on Flutter versions prior to `3.3`, or [`PlatformDispatcher.onError`](https://api.flutter.dev/flutter/dart-ui/PlatformDispatcher/onError.html) on Flutter versions 3.3 and higher, so that errors are automatically captured.
+
+If you need a custom error handling zone which also provides automatic error reporting and breadcrumb tracking, use `Sentry.runZonedGuarded`. It wraps Dart's native [`runZonedGuarded`](https://api.flutter.dev/flutter/dart-async/runZonedGuarded.html) function with Sentry-specific functionality.
+
+This function automatically records calls to `print()` as `Breadcrumbs` and can be configured using `SentryOptions.enablePrintBreadcrumbs`.
+
+```dart
+Sentry.runZonedGuarded(() {
+  WidgetsBinding.ensureInitialized();
+
+  // Errors before init will not be handled by Sentry
+
+  SentryFlutter.init(
+    (options) {
+    ...
+    },
+    appRunner: () => runApp(MyApp()),
```

Comment:
I wonder if we should also include a tab in the code snippet like `FlutterWeb with WidgetsBindingIntegration` or something like that in the getting started page because I think it's rather common to want to initialize the WidgetsBinding before the Sentry init and even though we add the info here, I would imagine it's still hard to find, the user would have to scroll through all the pages

but not sure

Reply:
Added it as a tab, not sure if we should add additional reference in the text, as it is also describe here in detail.